database.rules.json

```diff
@@ -0,0 +1,7 @@
+{
+  /* Visit https://firebase.google.com/docs/database/security to learn more about security rules. */
```

Comment:
this might be a problem because this is a json file. Comments aren't exactly `rad` in json